packages/blitz/src/cli/commands/generate.ts

```diff
@@ -72,6 +74,7 @@ const args = arg(
     "--parent": String,
     "--dry-run": Boolean,
     "--env": String,
+    "--generate-custom-templates": Boolean,
```

Comment:
I feel like we should use generator type for this instead of a flag, because flags are things that work with any command but this doesn't.

so instead like this: `blitz generate custom-templates`

Reply:
@flybayer yeah I agree, I have now removed the flag and converted it to a blitz generator command